src/gles_fns.rs

```diff
@@ -147,47 +147,102 @@ impl Gl for GlesFns {
 
     #[allow(unused_variables)]
     fn gen_queries(&self, n: GLsizei) -> Vec<GLuint> {
-        panic!("not supported")
+        if !self.ffi_gl_.GenQueriesEXT.is_loaded() {
+            return Vec::new();
+        }
+        unsafe {
+            let mut result = vec![0; n as usize];
+            self.ffi_gl_.GenQueriesEXT(n, result.as_mut_ptr());
```

Comment:
only this line needs to be inside `unsafe` block

Reply:
I will update it. I also going to update same one in gl_fns.rs.